clang/lib/CodeGen/TargetBuiltins/ARM.cpp

```diff
@@ -4918,6 +4919,31 @@ Value *CodeGenFunction::EmitAArch64SMEBuiltinExpr(unsigned BuiltinID,
   // Handle builtins which require their multi-vector operands to be swapped
   swapCommutativeSMEOperands(BuiltinID, Ops);
 
+  auto isCntsBuiltin = [&](int64_t &Mul) {
+    switch (BuiltinID) {
```

Comment:
nit: This could just return the `mul`, and the use below could be `if (auto mul = isCntsBuiltin()))`